tests/integration/test_xtce_based_parsing/test_inheritance_restrictions.py

```diff
@@ -9,6 +9,15 @@ def test_jpss_xtce_packet_parsing(jpss_test_data_dir):
     jpss_definition = definitions.XtcePacketDefinition.from_xtce(xtce_document=jpss_xtce)
     assert isinstance(jpss_definition, definitions.XtcePacketDefinition)
 
+    # Prove that parsed objects are referencing the same objects, not duplicates
```

Comment:
I thought you had this in your previous PR?

Reply:
Rebasing mistake. These assertions are in the unit tests for definitions.py. This just snuck back in when I rebased.